Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/ProjectedNodeTrait.php

```diff
@@ -241,6 +241,28 @@ public function iExpectTheNodeWithAggregateIdentifierToNotContainTheTag(string $
         });
     }
 
+    /**
+     * @Then /^I expect this node to be exactly explicitly tagged "(.*)"$/
+     * @param string $tagList the comma-separated list of tag names
+     */
+    public function iExpectThisNodeToBeExactlyExplicitlyTagged(string $tagList): void
+    {
+        $this->assertOnCurrentNode(function (Node $currentNode) use ($tagList) {
+            $currentNode->tags->withoutInherited()->toStringArray() === explode(',', $tagList);
```

Comment:
lol i just noticed this is a false positive test. There are actually no assertions here but just an expression and we return void.

`Assert::assertEquals` must be used for this test to have an effect i guess?

Or am i missing something?

Reply:
Good catch! o.O

Reply:
Thanks a lot!

This seems to be a real false positive, since there seem to be issues with MoveNode vs. Subtree tags. I'm currently investigating

Reply:
The fix for this is applied here:
https://github.com/neos/neos-development-collection/pull/5025